src/Entity/MetadataDisplayEntity.php

```diff
@@ -218,6 +228,17 @@ public static function baseFieldDefinitions(EntityTypeInterface $entity_type) {
       ->setDescription(t('The ID of the Metadata Display entity.'))
       ->setReadOnly(TRUE);
 
+    $fields['revision_id'] = BaseFieldDefinition::create('integer')
+      ->setLabel(t('REVISION_ID'))
+      ->setDescription(t('The revision ID of the Metadata Display entity.'))
+      ->setReadOnly(TRUE);
+    $fields['changed'] = BaseFieldDefinition::create('changed')
+      ->setLabel(t('Changed'))
+      ->setDescription(t('The time that the term was last edited.'))
```

Comment:
Not a term!

Reply:
Nothing is stupid, I appreciate the quick response and the learning path you have taken